aquila/urls.py

```diff
@@ -29,4 +31,8 @@
     path('groupings/<int:pk>/edit/', GroupingUpdateView.as_view(), name="groupings-update"),
     path('login/', AquilaLoginView.as_view(template_name="users/login.html"), name="login"),
     path('logout/', LogoutView.as_view(next_page="/login"), name="logout"),
+    path('rights/', RightsShellListView.as_view(), name='rights-list'),
+    path('rights/create/', RightsShellCreateView.as_view(), name='rights-create'),
+    path('rights/<int:pk>/update/', RightsShellUpdateView.as_view(), name='rights-update'),
+    path('rights/<int:pk>', RightsShellDetailView.as_view(), name='rights-detail'),
```

Comment:
```suggestion
    path('rights/<int:pk>/', RightsShellDetailView.as_view(), name='rights-detail'),
```
I think you need a trailing slash here.